snap/local/cleanup.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
Maybe you could reuse : https://github.com/ubuntu-robotics/snapcraft-ros-clean
no need to do it in this PR, maybe we can simply create an issue.